packages/atclient/src/atclient_utils.c

```diff
@@ -124,3 +124,29 @@ int atclient_utils_populate_atkeys_from_homedir(atclient_atkeys *atkeys, const c
 
 exit: { return ret; }
 }
+
+int atclient_utils_find_index_past_at_prompt(const unsigned char *read_buf, size_t read_n, size_t *read_i) {
```

Comment:
Moved from a private method in atclient.c to public in atclient_utils so that it can be called from monitor.